packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -174,6 +245,7 @@ SelectedFilters.propTypes = {
 	title: types.title,
 	onChange: types.func,
 	componentProps: types.props,
```

Comment:
@mohdashraf010897 I can see that `componentProps` is already being used here. I would recommend using the Context to read the store.
![alt img](https://i.imgur.com/MSbcpmX.png)

Reply:
👍  now using context